get-started/index.md

```diff
@@ -83,25 +83,25 @@ Continue to [Setup](#setup).
 
 ## [Setup](#setup)
 
-**Note:** An alternative setup using the multi-namespace **ClusterCryostat** `CR` is described in
-[Alternate Setup](#alternate-setup). For simplicity we will continue with the single-namespace **Cryostat** `CR`.
+**Note:** An alternative setup using the multi-namespace **ClusterCryostat CR** is described in
+[Alternate Setup](#alternate-setup). For simplicity we will continue with the single-namespace **Cryostat CR**.
 
 ### [Deploying Cryostat](#deploying-cryostat)
 
 {% include howto_step.html
   summary="Create a Cryostat instance"
   image-name="cryostat-operatorhub-install-complete.png"
 %}
-Once the installation is complete, click ***Create Cryostat*** to create a **Cryostat** `CR` instance in an **OpenShift
-Project** (Kubernetes `Namespace`) of your choice. This provides configuration information for the **Operator** to know the
+Once the installation is complete, click *Create Cryostat* to create a **Cryostat CR** instance in an **OpenShift
+Project** (**Kubernetes** `Namespace`) of your choice. This provides configuration information for the **Operator** to know the
```

Comment:
"**OpenShift Project**" vs "**Kubernetes** `Namespace`" is weird. OpenShift is a kind of Kubernetes, and what Kubernetes calls a Namespace, OpenShift calls a Project. So these terms should be treated similarly.

Reply:
I dint know this , thanks for the clarification.